iceoryx_hoofs/include/iceoryx_hoofs/design_pattern/builder.hpp

```diff
@@ -43,19 +43,24 @@
 ///     // END
 ///   };
 /// @endcode
-// NOLINTJUSTIFICATION brackets around macro parameter would lead in this case to compile time failures
+// AXIVION Next Construct AutosarC++19_03-A16.0.1 this macro is used to prevent a large amount of boilerplate code
+// which cannot be realized with templates or constexpr functions
+// AXIVION Next Construct AutosarC++19_03-A16.0.6 brackets around macro parameter would lead in this case to compile
```

Comment:
I can't find that in the AUTOSAR 19-03 document. You mean..?

```suggestion
// AXIVION Next Construct AutosarC++19_03-M16.0.6 brackets around macro parameter would lead in this case to compile
```

Reply:
doh :sweat: